Cargo.toml

```diff
@@ -54,6 +54,10 @@ status = "actively-developed"
 [badges.travis-ci]
 repository = "uuid-rs/uuid"
 
+[dependencies.getrandom]
```

Comment:
I see you are adding the dependency but you didn't remove `rand`. Is this intentional or just a miss?

Reply:
It's still needed for the `stdweb` and `wasm-bindgen` features.